demo/server.js

```diff
@@ -130,7 +130,6 @@ initSqlJs().then(SQL => {
 			data.transactionHash,
 			data.originalData.text,
 		])
-		// eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Why did you delete this line?

Reply:
automation lint fix did

Reply:
i thought you have added this feature in  fix also to delete eslint

Reply:
I'm confused. Are you saying that `yarn lint-fix` removed that line? It did not for me.

> I thought you have added this feature in fix also to delete eslint
What does that mean?

Please add the line back.

Reply:
okay
sure